src/wg/color/rgb/colorspace.d

```diff
@@ -268,6 +268,24 @@ immutable(RGBColorSpace)* findRGBColorspace(const(char)[] name) pure nothrow @no
     return null;
 }
 
+/**
+Returns the id of the color namespace with given parameters or empty array if parameters
+don't describe any standard color namespecase.
+*/
+const(char)[] rgbColorSpaceName(xyY white, float rx, float ry, float gx, float gy, float bx, float by) pure nothrow @nogc
+{
+    enum float epsilon = 0.001f;
+    foreach (ref def; rgbColorSpaceDefs)
+    {
+        if (def.white.x - white.x < epsilon && def.white.y - white.y < epsilon && def.white.Y - white.Y < epsilon && 
```

Comment:
did you consider using of `approxEqual` or like?

Reply:
It looks like you should take the absolute values?

Reply:
I added `fabs` around subtractions. I didn't know about approxEqual but after reading about it seems to me we don't need its complexity with relDiff and absDiff. Is that ok?

Reply:
Pretty much, relative difference is a divide, and that only makes sense for things with a large dynamic range (such as comparing say exp(x) and an approximation of exp(x))